lib/plugins/aws/configCredentials/awsConfigCredentials.test.js

```diff
@@ -2,32 +2,28 @@
 
 const expect = require('chai').expect;
 const sandbox = require('sinon');
-const constants = require('constants');
+const constants = require('fs').constants;
```

Comment:
Yak-shaving, but why not `const { constants } = require('fs')` or moving the following right after the `require` statements `const { constants } = fs`.